config.py

```diff
@@ -1,3 +1,24 @@
+import logging
+
+FORMAT = '%(asctime)s - %(name)-s - %(levelname)-s - %(message)s'
+LEVEL = logging.DEBUG
+logging.basicConfig(format=FORMAT, level=LEVEL)
+log = logging.getLogger(__name__)
+
+def logger(fn):
+    from functools import wraps
+    import inspect
+    @wraps(fn)
+    def wrapper(*args, **kwargs):
+        log = logging.getLogger(fn.__name__)
+        log.info('About to run %s' % fn.__name__)
+
+        out = apply(fn, args, kwargs)
+
+        log.info('Done running %s' % fn.__name__)
+        return out
+    return wrapper
+
 user = "root"
 password = "root"
 host = "localhost"
```

Comment:
Looks like you did not pull the latest config.py, pull the latest changes
Do not commit `config.py`

Reply:
I didn't get this @vishakha-lall , are you telling me to pull a single `config.py` file? And how can I not commit it? It would be really helpful if you clarify it. Thanks :)

Reply:
Instructions to not commit the `comfig.py` have been shared in the instructions in the [readme](https://github.com/vishakha-lall/MapBot#how-to-set-me-up). The config.py file was never meant to be committed. It had to be a part of `.gitignore`. Did you miss this step?
@janakrajchadha we would need to revert commits to undo this, right?

Reply:
Oh, I guess I missed this step. Really sorry for it. Please let me know what should I do to fix this?
Should I revert commits to undo this?